### DISCUSSION TITLE
Continuous POMDP Problem and Solver Example

### DISCUSSION THREAD

**Discussion by Omastto1** [JuliaPOMDP/POMDPs.jl#352] [Q&A]
Hi,

I am getting to know continuous POMDPs, and I wonder whether I can find some tutorial-like examples in this library.

I believe this to be the right place for this information or discussion about moving forward with that idea.

Tom

**Comment by zsunberg** [comment]
Hi Tom,

I would say that we don't really have a detailed *tutorial* on continuous POMDPs at the moment, but you might find some inspiration for problems in the POMDP Gallery: https://github.com/JuliaPOMDP/POMDPGallery.jl

There are a lot of different things that you might mean in a tutorial. For instance, are you looking for a tutorial about *defining* a continuous POMDP, or *solving* a pre-defined one? Also, "continuous POMDP" can have a lot of different meanings. Are all of the State, Action, and Observation continuous, or are some of them discrete? Some people also mean continuous-time when they say "Continuous POMDP".
